Ghidra/Features/Base/src/main/java/ghidra/program/flatapi/FlatProgramAPI.java

```diff
@@ -1514,6 +1514,58 @@ public final List<Symbol> getSymbols(String name, Namespace namespace) {
 	public final Namespace getNamespace(Namespace parent, String namespaceName) {
 		return currentProgram.getSymbolTable().getNamespace(namespaceName, parent);
 	}
+	
+	/**
+	 * Creates a new namespace with the given name contained inside the
+	 * specified parent namespace.
+	 * Pass <code>null</code> for namespace to indicate the global namespace.
+	 * If a Namespace or GhidraClass with the given name already exists, the
```

Comment:
Suggest creating links for Namespace and GhidraClass

Reply:
Will do. I actually had them initially but then removed them thinking it was overkill.

Reply:
I did a few of them but wasn't sure about doing it for all of them. If you think it makes sense I will make all of them a link.